### DISCUSSION TITLE
Prevent "m_" for script class exported member variables from showing in editor

### DISCUSSION THREAD

**Discussion by Vaani-del** [godotengine/godot-proposals#4523] [Editor]
Hi, in my scripts I use "m_" for my member variables (I use C#, not sure how this affects other languages). When I have a variable declared with "[Export]", it appears in the editor like the top variable on the attached image. It would be cleaner (and nicer) if there was an option to ignore the "m_".

Unity's [SerializeField] variables for example, will show it as the second variable in the attached screenshot, and it looks so much better as a result.

![image](https://user-images.githubusercontent.com/8584459/167531654-4a5bd220-78d5-4671-af42-ddc77f03267c.png)

**Comment by Calinou** [comment]
I'm not sure if removing one-letter prefixes with no way to opt out is a good idea. Instead, we may want to add an annotation to specify an arbitrary property name to expose: https://github.com/godotengine/godot-proposals/issues/3510
